e2e/cleanup_test.go

```diff
@@ -276,4 +274,14 @@ spec:
 		stdout, stderr, err := kubectl("delete", "ns", cleanupTest)
 		Expect(err).ShouldNot(HaveOccurred(), "stdout=%s, stderr=%s", stdout, stderr)
 	})
+
+	AfterSuite(func() {
```

Comment:
Sorry, my suggestion was a mistake.
`AfterSuite` is a global teardown, so it should be used at the top level.
In this case, I think calling umount and lvremove directly is sufficient.

Reply:
Fmm, it seems each `It` is independent.
https://github.com/onsi/ginkgo/issues/70#issuecomment-42882976
So, the current test is broken...?

Reply:
By Ginkgo default, `It` runs on sequence order.
@bells17 I think the first implementation is better, would you remove this [commit](https://github.com/topolvm/topolvm/pull/230/commits/2bf86f4aec7eba7d1c58c95b9e5d2a4403af1105)?

Reply:
fixed it.